templates/style/_themes.scss

```
@@ -0,0 +1,25 @@
+// Standard white theme
+html {
+  --color-background-code: #f5f5f5;
+  --color-background: #fff;
+  --color-border-light: lighten(#ddd, 5%);
```

Should probably be 
```suggestion
  --color-border-light: lighten(var(--color-border), 5%);
```
although not sure if that syntax works.

---

Nope, `lighten` is an SCSS function which needs the value at compile time, _but_ lucky you mentioned it cause it turns out that function doesn't run in this context for some reason and instead `lighten` is put into the output CSS, I think we need to just precalculate the values here.